### PR TITLE
Show the map on the course details page when query param is set

### PR DESCRIPTION
**Pull request by tvararu**
Users can set `?previewMap=true` to see it.

### CODE REVIEW

File: shared/Views/Shared/Components/CourseDetails/_Apply.cshtml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 @{
   var show2019Apply = FeatureFlags.Apply2019;
-  var showMap = FeatureFlags.Maps;
+  var showMap = ViewBag.showMap;
```
Comment (fofr) [inline shared/Views/Shared/Components/CourseDetails/_Apply.cshtml]: Why is one showMap and one previewMap?

Comment (tvararu) [reply shared/Views/Shared/Components/CourseDetails/_Apply.cshtml]: My reasoning is that `showMap` is what the variable does, its effect. `previewMap` is what we tell our users to append to the URL, and thus communicates to them that they are looking at a preview of an upcoming feature. `?showMap=true` would also work but it wouldn't carry that meaning.

File: src/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -39,18 +40,34 @@
...
+=======
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+    ga('create', 'UA-112932657-1', 'auto');
+    ga('set', 'transport', 'beacon');
+    ga('set', 'anonymizeIp', true);
+>>>>>>> Stashed changes
```
Comment (timabell) [inline src/Views/Shared/_Layout.cshtml]: you committed a merge conflict

File: src/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -39,18 +40,34 @@
...
+    }
 
+<<<<<<< Updated upstream
       ga('send', 'pageview');
     </script>
+=======
+    ga('send', 'pageview');
+  </script>
+}
+>>>>>>> Stashed changes
```
Comment (timabell) [inline src/Views/Shared/_Layout.cshtml]: and another

Comment (tvararu) [reply src/Views/Shared/_Layout.cshtml]: Huh wow

**Pull request merged**